snap/info_snap_yaml.go

```diff
@@ -78,6 +78,7 @@ type appYaml struct {
 	ReloadCommand   string          `yaml:"reload-command,omitempty"`
 	PostStopCommand string          `yaml:"post-stop-command,omitempty"`
 	StopTimeout     timeout.Timeout `yaml:"stop-timeout,omitempty"`
+	StartTimeout    timeout.Timeout `yaml:"start-timeout,omitempty"`
```

Comment:
A side note, we have validation for WatchdogTimeout (>0), if you feel like it, the PR could be extended to add validation for (Start|Stop)Timeout too.